packages/expect/src/index.js

```diff
@@ -314,19 +314,30 @@ const _validateResult = result => {
   }
 };
 
+function assertions(expected: number) {
+  const error = new Error();
+  Error.captureStackTrace(error, assertions);
```

Comment:
shouldn't we try/catch this for non V8 engines?

Reply:
ah, good point